acl.go

```diff
@@ -277,8 +277,10 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
 	case string:
 		severity = cacheACL.Fields["severity"].(string)
 	case libovsdb.OvsSet:
-		for _, a := range cacheACL.Fields["meter"].(libovsdb.OvsSet).GoSet {
-			severity = a.(string)
+		if meters, ok := cacheACL.Fields["meter"]; ok {
```

Comment:
Thanks for catching this. @fyuan1316 Can you also do this check for severity and other meter usage in rowToACL function? Just move this if meters, ok := cacheACL.Fields["meter"]; ok  before swith case https://github.com/eBay/go-ovn/blob/master/acl.go#L265